Version 4.0.1 stable, boostrap glyphicons don't seem to work

- [x] I understand that GitHub issues are not for tech support, but for questions specific to this generator, bug reports, and feature requests.

| Item | Version |
| --- | --- |
| generator-angular-fullstack | 4.0.1 |
| Node | 4.2.6 |
| npm | 3.5.2 |
| Operating System | Ubuntu 16.04 LTS |
| etc | etc |

| Item | Answer |
| --- | --- |
| Transpiler | Babel |
| Markup | HTML |
| CSS | CSS |
| Router | ui-router |
| Client Tests | Mocha |
| DB | MongoDB |
| Auth | Y |

After I run npm install, and gulp serve the glypicons don't seem to work. They are just missing/square characters. I tried both in chrome and firefox. 

If I access the font files from the browser like this they get downloaded:

localhost:3000/assets/fonts/bootstrap/glyphicons-halflings-regular.svg 

However I don't understand why they are not showing up correctly in the browser.

I get some errors when i'm starting the apps from gulp. I don't know if they are of any relevance.

`[11:13:27] Starting 'serve'...
[11:13:27] Starting 'clean:tmp'...
[11:13:27] Starting 'lint:scripts'...
[11:13:27] Starting 'lint:scripts:client'...
[11:13:27] Starting 'lint:scripts:server'...
[11:13:27] Starting 'inject'...
[11:13:27] Starting 'inject:css'...
[11:13:27] Starting 'copy:fonts:dev'...
[11:13:27] Starting 'env:all'...
[11:13:27] Finished 'env:all' after 149 ms
[11:13:27] Finished 'clean:tmp' after 532 ms
[11:13:31] gulp-inject 5 files into app.css.
[11:13:31] Finished 'inject:css' after 3.97 s
[11:13:31] Finished 'inject' after 3.97 s
[11:13:34] Finished 'copy:fonts:dev' after 7.02 s
[11:13:35] 
~/server/app.js
  16:17  warning  Unexpected string concatenation                                                                                prefer-template
  17:3   error    Don't use process.exit(); throw an error instead                                                               no-process-exit
  21:1   error    Unexpected space(s) after "if"                                                                                 keyword-spacing
  21:22  error    Statement inside of curly braces should be on next line                                                        brace-style
  21:22  error    Closing curly brace should be on the same line as opening curly brace or on the line after the previous block  brace-style

~/server/index.js
  4:11  warning  Unexpected use of process.env   no-process-env
  4:34  warning  Unexpected use of process.env   no-process-env
  6:1   error    Unexpected space(s) after "if"  keyword-spacing

~/server/routes.js
  24:33  warning  Unexpected string concatenation  prefer-template

~/server/auth/auth.service.js
   3:8   error    'passport' is defined but never used                    no-unused-vars
  23:7   error    Unexpected space(s) after "if"                          keyword-spacing
  24:37  warning  Unexpected string concatenation                         prefer-template
  27:6   error    Unexpected space(s) after "if"                          keyword-spacing
  27:6   error    Expected indentation of 6 space characters but found 5  indent
  28:36  warning  Unexpected string concatenation                         prefer-template
  36:11  error    Unexpected space(s) after "if"                          keyword-spacing
  50:3   error    Unexpected space(s) after "if"                          keyword-spacing
  57:7   error    Unexpected space(s) after "if"                          keyword-spacing
  57:53  warning  '>=' should be placed at the beginning of the line      operator-linebreak
  59:9   error    Expected return with your callback function             callback-return
  70:30  warning  Expected property shorthand                             object-shorthand
  79:3   error    Unexpected space(s) after "if"                          keyword-spacing

~/server/auth/index.js
  4:8  error  'passport' is defined but never used  no-unused-vars

~/server/config/express.js
   27:3   error    Unexpected space(s) after "if"                            keyword-spacing
   31:3   error    Unexpected space(s) after "if"                            keyword-spacing
   39:20  warning  Unexpected string concatenation                           prefer-template
   49:1   error    Trailing spaces not allowed                               no-trailing-spaces
   67:3   error    Unexpected space(s) after "if"                            keyword-spacing
   67:26  warning  Unexpected use of process.env                             no-process-env
   82:3   error    Unexpected space(s) after "if"                            keyword-spacing
   82:7   error    Expected literal to be on the right side of ===           yoda
   84:45  error    Trailing spaces not allowed                               no-trailing-spaces
   89:58  error    Trailing spaces not allowed                               no-trailing-spaces
   97:14  warning  Unexpected string concatenation                           prefer-template
  105:26  error    Trailing spaces not allowed                               no-trailing-spaces
  117:37  error    Unexpected space before function parentheses              space-before-function-paren
  119:9   error    Expected indentation of 6 space characters but found 8    indent
  119:9   error    Unexpected space(s) after "if"                            keyword-spacing
  120:13  error    Expected indentation of 10 space characters but found 12  indent
  121:17  error    Expected indentation of 14 space characters but found 16  indent
  121:24  error    Strings must use singlequote                              quotes
  122:17  error    Expected indentation of 14 space characters but found 16  indent
  123:17  error    Expected indentation of 14 space characters but found 16  indent
  126:9   error    Expected indentation of 6 space characters but found 8    indent
  130:3   error    Unexpected space(s) after "if"                            keyword-spacing
  130:7   error    Expected literal to be on the right side of ===           yoda
  130:32  error    Expected literal to be on the right side of ===           yoda

~/server/config/seed.js
  14:97  warning  '+' should be placed at the beginning of the line  operator-linebreak
  15:78  warning  '+' should be placed at the beginning of the line  operator-linebreak
  19:72  warning  '+' should be placed at the beginning of the line  operator-linebreak
  23:73  warning  '+' should be placed at the beginning of the line  operator-linebreak
  24:75  warning  '+' should be placed at the beginning of the line  operator-linebreak
  28:75  warning  '+' should be placed at the beginning of the line  operator-linebreak
  32:78  warning  '+' should be placed at the beginning of the line  operator-linebreak
  33:80  warning  '+' should be placed at the beginning of the line  operator-linebreak
  37:79  warning  '+' should be placed at the beginning of the line  operator-linebreak

~/server/config/socketio.js
   6:8   error    'config' is defined but never used                 no-unused-vars
   9:23  error    'socket' is defined but never used                 no-unused-vars
  22:1   error    Block must not be padded by blank lines            padded-blocks
  41:22  warning  Unexpected string concatenation                    prefer-template
  41:63  warning  '+' should be placed at the beginning of the line  operator-linebreak

~/server/api/thing/thing.controller.js
   29:7  warning  Avoid using Function.prototype.apply, instead use Reflect.apply     prefer-reflect
   91:5  warning  Avoid using the delete keyword, instead use Reflect.deleteProperty  prefer-reflect
  102:5  warning  Avoid using the delete keyword, instead use Reflect.deleteProperty  prefer-reflect

~/server/api/thing/thing.events.js
  28:22  warning  Unexpected string concatenation  prefer-template

~/server/api/user/user.controller.js
    4:8   error  'passport' is defined but never used  no-unused-vars
   12:4   error  Missing semicolon                     semi
   37:34  error  'next' is defined but never used      no-unused-vars
   59:7   error  Unexpected space(s) after "if"        keyword-spacing
   82:42  error  'next' is defined but never used      no-unused-vars
   89:7   error  Unexpected space(s) after "if"        keyword-spacing
  110:7   error  Unexpected space(s) after "if"        keyword-spacing
  121:40  error  'next' is defined but never used      no-unused-vars

~/server/api/user/user.events.js
  16:3   error    Unnecessarily quoted property 'save' found      quote-props
  17:3   error    Unnecessarily quoted property 'remove' found    quote-props
  21:1   error    Unexpected space(s) after "for"                 keyword-spacing
  26:20  error    'event' is already declared in the upper scope  no-shadow
  28:21  warning  Unexpected string concatenation                 prefer-template
  30:4   error    Missing semicolon                               semi

~/server/api/user/user.model.js
   28:5   warning  Expected method shorthand                       object-shorthand
  193:15  warning  Use the rest parameters instead of 'arguments'  prefer-rest-params
  194:18  warning  Use the rest parameters instead of 'arguments'  prefer-rest-params
  196:22  warning  Use the rest parameters instead of 'arguments'  prefer-rest-params
  197:18  warning  Use the rest parameters instead of 'arguments'  prefer-rest-params
  237:14  warning  Unexpected sync method: 'pbkdf2Sync'            no-sync
  243:9   error    Expected return with your callback function     callback-return
  245:9   error    Expected return with your callback function     callback-return

✖ 87 problems (54 errors, 33 warnings)

[11:13:35] Finished 'lint:scripts:server' after 8.33 s
[11:13:35] 
~/client/polyfills.js
  15:1  error  Unexpected space(s) after "if"  keyword-spacing
  19:1  error  Unexpected space(s) after "if"  keyword-spacing

~/client/app/app.js
  34:5  error  Expected indentation of 2 space characters but found 4  indent
  38:5  error  Expected indentation of 2 space characters but found 4  indent
  39:3  error  Expected indentation of 0 space characters but found 2  indent
  47:9  error  Unexpected space(s) after "if"                          keyword-spacing

~/client/app/account/account.routes.js
   7:7  error    Expected indentation of 4 space characters but found 6  indent
   8:7  error    Expected indentation of 4 space characters but found 6  indent
   9:7  error    Expected indentation of 4 space characters but found 6  indent
  10:7  error    Expected indentation of 4 space characters but found 6  indent
  11:5  error    Expected indentation of 2 space characters but found 4  indent
  16:7  warning  Expected method shorthand                               object-shorthand

~/client/app/account/index.js
  21:7  error  Unexpected space(s) after "if"  keyword-spacing

~/client/app/admin/admin.routes.js
  13:2  error  Unnecessary semicolon  no-extra-semi

~/client/app/admin/index.js
  6:16  error  'angular' is not defined  no-undef

~/client/app/main/main.component.js
  26:5   error    Unexpected space(s) after "if"   keyword-spacing
  35:23  warning  Unexpected string concatenation  prefer-template

~/client/app/main/main.routes.js
  10:2  error  Unnecessary semicolon  no-extra-semi

~/client/components/auth/auth.service.js
    3:7   error    'User' is defined but never used                                 no-unused-vars
    5:78  error    'User' is already declared in the upper scope                    no-shadow
   20:3   error    Unexpected space(s) after "if"                                   keyword-spacing
   38:11  error    Expected indentation of 8 space characters but found 10          indent
   39:11  error    Expected indentation of 8 space characters but found 10          indent
   40:9   error    Expected indentation of 6 space characters but found 8           indent
   74:11  error    Expected indentation of 8 space characters but found 10          indent
   75:11  error    Expected indentation of 8 space characters but found 10          indent
   76:11  error    Expected indentation of 8 space characters but found 10          indent
   77:9   error    Expected indentation of 6 space characters but found 8           indent
   94:11  error    Expected indentation of 8 space characters but found 10          indent
   95:9   error    Expected indentation of 6 space characters but found 8           indent
  193:14  warning  Avoid using Function.prototype.apply, instead use Reflect.apply  prefer-reflect
  193:14  warning  use the spread operator instead of the '.apply()'                prefer-spread
  193:39  warning  Avoid using Function.prototype.apply, instead use Reflect.apply  prefer-reflect
  193:66  warning  Use the rest parameters instead of 'arguments'                   prefer-rest-params
  202:14  warning  Unexpected sync method: 'hasRoleSync'                            no-sync

~/client/components/auth/interceptor.service.js
  11:7   error    Unexpected space(s) after "if"           keyword-spacing
  12:40  warning  Unexpected string concatenation          prefer-template
  19:7   error    Unexpected space(s) after "if"           keyword-spacing
  19:36  error    Block must not be padded by blank lines  padded-blocks

~/client/components/auth/router.decorator.js
   3:13  error  '_' is defined but never used            no-unused-vars
  10:5   error  Unexpected space(s) after "if"           keyword-spacing
  14:5   error  Unexpected space(s) after "if"           keyword-spacing
  17:11  error  Unexpected space(s) after "if"           keyword-spacing
  23:25  error  Block must not be padded by blank lines  padded-blocks
  31:11  error  Unexpected space(s) after "if"           keyword-spacing
  41:2   error  Unnecessary semicolon                    no-extra-semi

~/client/components/modal/modal.service.js
  13:5   error    'angular' is not defined                                         no-undef
  33:20  error    'angular' is not defined                                         no-undef
  40:11  error    Split 'var' declarations into multiple statements                one-var
  40:22  warning  Avoid using Function.prototype.call, instead use Reflect.apply   prefer-reflect
  40:49  warning  Use the rest parameters instead of 'arguments'                   prefer-rest-params
  48:21  warning  Unexpected string concatenation                                  prefer-template
  52:17  warning  Expected method shorthand                                        object-shorthand
  58:17  warning  Expected method shorthand                                        object-shorthand
  66:13  warning  Avoid using Function.prototype.apply, instead use Reflect.apply  prefer-reflect
  74:16  error    'angular' is not defined                                         no-undef

~/client/components/mongoose-error/mongoose-error.directive.js
   7:1  error    'angular' is not defined   no-undef
  12:7  warning  Expected method shorthand  object-shorthand

~/client/components/navbar/navbar.component.js
   8:23  warning  Unexpected sync method: 'isLoggedInSync'      no-sync
   9:20  warning  Unexpected sync method: 'isAdminSync'         no-sync
  10:27  warning  Unexpected sync method: 'getCurrentUserSync'  no-sync
  15:16  error    'angular' is not defined                      no-undef

~/client/components/oauth-buttons/index.js
  5:29  warning  Unexpected string concatenation  prefer-template
  9:16  error    'angular' is not defined         no-undef

~/client/components/socket/socket.service.js
  31:18  error    'angular' is not defined         no-undef
  36:17  warning  Unexpected string concatenation  prefer-template
  37:23  error    '_' is not defined               no-undef
  45:9   error    Unexpected space(s) after "if"   keyword-spacing
  58:17  warning  Unexpected string concatenation  prefer-template
  60:9   error    '_' is not defined               no-undef
  73:33  warning  Unexpected string concatenation  prefer-template
  74:33  warning  Unexpected string concatenation  prefer-template
  79:16  error    'angular' is not defined         no-undef

~/client/components/util/util.module.js
  7:16  error  'angular' is not defined  no-undef

~/client/components/util/util.service.js
  18:14  error    'angular' is not defined                             no-undef
  18:44  error    'angular' is not defined                             no-undef
  32:7   error    Unexpected space(s) after "if"                       keyword-spacing
  58:99  warning  '===' should be placed at the beginning of the line  operator-linebreak

~/client/app/account/login/index.js
  5:16  error  'angular' is not defined  no-undef

~/client/app/account/login/login.controller.js
  14:5   error  Unexpected space(s) after "if"                           keyword-spacing
  16:11  error  Expected indentation of 8 space characters but found 10  indent
  17:11  error  Expected indentation of 8 space characters but found 10  indent
  18:9   error  Expected indentation of 6 space characters but found 8   indent

~/client/app/account/settings/index.js
  5:16  error  'angular' is not defined  no-undef

~/client/app/account/settings/settings.controller.js
  13:5  error  Unexpected space(s) after "if"  keyword-spacing

~/client/app/account/signup/index.js
  5:16  error  'angular' is not defined  no-undef

~/client/app/account/signup/signup.controller.js
  14:5   error  Unexpected space(s) after "if"                           keyword-spacing
  16:11  error  Expected indentation of 8 space characters but found 10  indent
  17:11  error  Expected indentation of 8 space characters but found 10  indent
  18:11  error  Expected indentation of 8 space characters but found 10  indent
  19:9   error  Expected indentation of 6 space characters but found 8   indent
  28:11  error  'angular' is not defined                                 no-undef

✖ 92 problems (68 errors, 24 warnings)

[11:13:35] Finished 'lint:scripts:client' after 8.56 s
[11:13:35] Finished 'lint:scripts' after 8.56 s
[11:13:35] Starting 'start:server'...
[11:13:35] Finished 'start:server' after 17 ms
[11:13:35] Starting 'start:client'...
[nodemon] 1.10.2`



I don't get any errors in the browser console however.

--- a/templates/app/webpack.make.js
+++ b/templates/app/webpack.make.js
@@ -211,7 +211,7 @@ module.exports = function makeWebpackConfig(options) {
                 //
                 // Reference: https://github.com/webpack/style-loader
                 // Use style-loader in development for hot-loading
-                ? ExtractTextPlugin.extract('style', 'css?sourceMap!postcss')
+                ? ExtractTextPlugin.extract('style', 'css!postcss')
                 // Reference: https://github.com/webpack/null-loader
                 // Skip loading css in test mode
                 : 'null'

